OpenSim/Simulation/Model/ConditionalPathPoint.h

```diff
@@ -50,6 +50,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(ConditionalPathPoint, PathPoint);
         "The minimum and maximum values that the coordinate can range between, "
         "for which the PathPoint is active. Angular coordinates in radians.");
 
+//==============================================================================
+// CONNECTORS
+//==============================================================================
+    OpenSim_DECLARE_CONNECTOR(coordinate, Coordinate,
+        "The coordinate whose value this path point depends on.");
```

Comment:
Or 
`The coordinate whose value determines when the path point is active according to the specified range.`

Either is fine with me or anything similar.